app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/NightscoutUploader.java

```diff
@@ -489,7 +489,7 @@ private boolean doRESTUpload(SharedPreferences prefs, List<BgReading> glucoseDat
                     last_success_time = JoH.tsl();
                     last_exception_count = 0;
                 } catch (Exception e) {
-                    String msg = "Unable to do REST API Upload: " + e.getMessage() + " url: " + baseURI + " marking record: " + (any_successes ? "succeeded" : "failed");
+                    String msg = "Unable to do REST API Upload: " + e.getMessage() + " url: " + " marking record: " + (any_successes ? "succeeded" : "failed");
```

Comment:
Can you remove the "url:" section in the raw string here?

Reply:
Thanks for the review.
I intentionally left it.  I just wanted to remove the url itself and nothing else.   I thought leaving the url acronym may serve as a hint that's where the problem is.

What would be the problem with leaving it?

Reply:
As-is it looks pretty confusing, because the syntax of how the message is worded implies that it's going to put something there. Reading this out of context I'd probably assume the URL was blank, versus that the URL was omitted intentionally.

I would recommend changing it to something like

```
"Unable to do REST API Upload. Check the configured Nightscout URL in the xDrip settings: " + e.getMessage() + " marking record: " + (any_successes ? "succeeded" : "failed");
```

Reply:
Removed "url:"
Thanks